pandas/tests/io/parser/test_common.py

```diff
@@ -1168,6 +1187,7 @@ def test_warn_if_chunks_have_mismatched_type(all_parsers):
     assert df.a.dtype == object
 
 
+@skip_pyarrow
```

Comment:
not all combinations fail, this only has two so should really add marker to pytest.mark.parametrize